x11-misc/xflux-gui/xflux-gui-1.1.10_p20180910.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+MY_PN="fluxgui"
```

Comment:
This is used only once, so you can inline it into MY_P.

Reply:
Ok, changed. But I liked it the other way more ;-)